apps/client/src/common/utils/deviceUtils.ts

```diff
@@ -3,6 +3,6 @@ export function isMacOS() {
   return userAgent.includes('macintosh') || userAgent.includes('mac os');
 }
 
-export const deviceAlt = isMacOS() ? '⌥' : 'Alt';
+export const deviceAlt = 'Alt';
```

Comment:
export const deviceAlt = isMacOS() ? 'Option' : 'Alt';

Reply:
Good catch, thank you!